<launch>
  <arg name="is_classic" default="true" />

  <node pkg="odml_data_processing" type="data_processing_node" name="data_processing_node" output="screen">
        <param name="rosbag_rate" type="double" value="0.33" />
  </node>
  
  <group if="$(arg is_classic)">
    <include file="$(find odml_visual_odometry)/launch/visual_odometry_classic.launch"/>
  </group>
  <group unless="$(arg is_classic)">
    <include file="$(find odml_visual_odometry)/launch/visual_odometry_superpoint.launch"/>
  </group>


  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find odml_data_processing)/rviz/main.rviz" />

</launch>